examples/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.16.3)
```

Comment:
Should we maybe wait for an example before adding this? There doesn't seem to be any source file yet.